src/System.Management.Automation/engine/InternalCommands.cs

```diff
@@ -295,12 +295,12 @@ protected override void ProcessRecord()
                                 else
                                 {
                                     // we write null out because:
-                                    // PS C:\> �$null | ForEach-object {$_.aa} | ForEach-Object {$_ + 3}�
+                                    // PS C:\> "$null | ForEach-object {$_.aa} | ForEach-Object {$_ + 3}"
```

Comment:
adding quotes here changes the meaning of that comment line from a powershell pipeline to a string